packages/application-generic/src/utils/filter.ts

```diff
@@ -23,31 +24,31 @@ export abstract class Filter {
     const filterValue = this.parseValue(actualValue, fieldFilter.value);
     let result = false;
 
-    if (fieldFilter.operator === 'EQUAL') {
+    if (fieldFilter.operator === FieldOperatorEnum.EQUAL) {
       result = actualValue === filterValue;
     }
-    if (fieldFilter.operator === 'NOT_EQUAL') {
+    if (fieldFilter.operator === FieldOperatorEnum.NOT_EQUAL) {
       result = actualValue !== filterValue;
     }
-    if (fieldFilter.operator === 'LARGER') {
+    if (fieldFilter.operator === FieldOperatorEnum.LARGER) {
       result = actualValue > filterValue;
     }
-    if (fieldFilter.operator === 'SMALLER') {
+    if (fieldFilter.operator === FieldOperatorEnum.SMALLER) {
       result = actualValue < filterValue;
     }
-    if (fieldFilter.operator === 'LARGER_EQUAL') {
+    if (fieldFilter.operator === FieldOperatorEnum.LARGER_EQUAL) {
       result = actualValue >= filterValue;
     }
-    if (fieldFilter.operator === 'SMALLER_EQUAL') {
+    if (fieldFilter.operator === FieldOperatorEnum.SMALLER_EQUAL) {
       result = actualValue <= filterValue;
     }
-    if (fieldFilter.operator === 'NOT_IN') {
+    if (fieldFilter.operator === FieldOperatorEnum.NOT_IN) {
       result = !actualValue.includes(filterValue);
     }
-    if (fieldFilter.operator === 'IN') {
+    if (fieldFilter.operator === FieldOperatorEnum.IN) {
       result = actualValue.includes(filterValue);
     }
-    if (fieldFilter.operator === 'IS_DEFINED') {
+    if (fieldFilter.operator === FieldOperatorEnum.IS_DEFINED) {
```

Comment:
I have noticed in this method `ALL_IN` functionality is not implemented. I ignore which was the reason for that.